### PR TITLE
Merge docs about migrating from Bolt 3 to Bolt 5 into 5.0

### PR DESCRIPTION
**Pull request by nestordedios**

### CODE REVIEW

File: docs/upgrading/upgrading-from-30.md [hunk 1]
```diff
@@ -1,14 +1,230 @@
...
-Upgrading from Bolt 3.0
+Migrating from Bolt v3.7 to v5.0
 =========================
 
-Bolt 4 is significantly different than Bolt 2 and Bolt 3. As such, there's no
-simple upgrade. In the near future, this page will have a migration guide, on
-how to set up a fresh Bolt 4 site, and how to migrate your content and/or theme
-to Bolt 4.
+Recently it has been announced the release of Bolt 5 and since Bolt 3 is reaching
+end of life soon, we have made this guide to make the migration process a bit more
```
Comment (I-Valchev) [inline docs/upgrading/upgrading-from-30.md]: @nestordedios I'd replace "soon" with the date on which it won't be supported anymore (see https://boltcms.io/newsitem/major-announcements-bolt-3-eol-bolt-4-2-5-0-releases)

That way the text will still be valid if we don't update it after the end of life. :-)

File: docs/upgrading/upgrading-from-30.md [hunk 2]
```diff
@@ -1,14 +1,230 @@
...
+```yaml
+# Bolt 3 Integer field definition
+integerfield:
+    type: integer
+
+# Bolt 3 Float field definition
+floatfield:
+    type: float
+```
+These fields do no longer exist in Bolt 5. This types of fields have been combined into one Field called
```
Comment (I-Valchev) [inline docs/upgrading/upgrading-from-30.md]: `these` types

File: docs/upgrading/upgrading-from-30.md [hunk 3]
```diff
@@ -1,14 +1,230 @@
...
+To migrate the data of an **Integer** field the new field definition will look like:
+```yaml
+# Bolt 5 float Number field definition
+floatfield:
+    type: number
+    mode: float 
+```
+
+### Repeater and Block fields
+The Reapeater and Block fields do no longer exist in Bolt 5. The Collection and Set field have been
```
Comment (I-Valchev) [inline docs/upgrading/upgrading-from-30.md]: `The Collection and Set fields`

File: docs/upgrading/upgrading-from-30.md [hunk 4]
```diff
@@ -1,14 +1,230 @@
...
+introduced to take over the role the other fields played. Even though the look and feel is pretty
+much the same as it was in earlier versions, the syntax to define these fields has differ a little bit.
```
Comment (I-Valchev) [inline docs/upgrading/upgrading-from-30.md]: `has differ` -> `differs`

File: docs/upgrading/upgrading-from-30.md [hunk 5]
```diff
@@ -1,14 +1,230 @@
...
+
+### Conimex Extension installation
+Install the extension by running:
+
+```shell script
+    composer require bobdenOtter/conimex
+```
+
+### Configuring the database
+Database configuration is done completely different from Bolt 3. You no longer need to add
```
Comment (I-Valchev) [inline docs/upgrading/upgrading-from-30.md]: `differently`

**Review by I-Valchev — COMMENTED** [review]
👍 Nice progress so far, and very easy to follow! @nestordedios

File: docs/upgrading/upgrading-from-30.md [hunk 6]
```diff
@@ -1,14 +1,230 @@
...
+
+### Moving Bolt 3 files into Bolt 5 installation
+Once you have installed the latest version of Bolt 5 move forward by copying relevant
+files of yout Bolt 3 site into the recently installed version 5.
+In the table below you can see an overview of the list of directories and files to be copied and  
+the location in the new version's structure.
+
+| Bolt 3                                | Bolt 5                                      |
+| ------------------------------------- | -------------------------------------------- |
+| General config directory `app/config`               | `config/bolt`                  |
```
Comment (UtechtDustin) [inline docs/upgrading/upgrading-from-30.md:98]: Config endings has changed from `yml` to `yaml`.

**Comment by Mikescops** [comment]
Crystal clear, thanks for this great documentation!

File: docs/upgrading/upgrading-from-30.md (hunk 1 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md]: Yes @I-Valchev , this is a better way to put. Thank you. 👍 

I have used the end of life date as you mentioned

File: docs/upgrading/upgrading-from-30.md (hunk 2 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md]: Yes, wrong orthography is corrected.

File: docs/upgrading/upgrading-from-30.md (hunk 3 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md]: This wrong orthography is corrected

File: docs/upgrading/upgrading-from-30.md (hunk 4 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md]: This is changed and it makes more sense now

File: docs/upgrading/upgrading-from-30.md (hunk 5 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md]: This is also fixed 👍

**Comment by nestordedios** [comment]
You are welcome @Mikescops. 

It's almost ready to be published.

File: docs/upgrading/upgrading-from-30.md (hunk 6 above)
Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md:98]: I'm not referring to any `.yml` file on the table. What do you mean?

Comment (nestordedios) [reply docs/upgrading/upgrading-from-30.md:98]: @UtechtDustin

**Review by bobdenotter — APPROVED** [review]

**Comment by I-Valchev** [comment]
Wow this is truly awesome! A lot of people in our community have mentioned it, and great to have it finally up! Nice work, @nestordedios !

**Pull request merged**